#Details of service data types
float64 changeASpeed
---
bool response
